testsuite/ada_lsp/S314-015.refactor_move_parameter_0/test.json

```diff
@@ -309,30 +309,7 @@
             {
                "id": 4,
                "result": [
-                  {
-                     "title": "Remove parameter A",
-                     "kind": "refactor.rewrite",
-                     "command": {
-                        "title": "",
-                        "command": "als-refactor-remove-parameters",
-                        "arguments": [
-                           {
-                              "context": "Default",
-                              "where": {
-                                 "textDocument": {
-                                    "uri": "$URI{main.adb}"
-                                 },
-                                 "position": {
-                                    "line": 1,
-                                    "character": 14
-                                 }
-                              },
-                              "first_parameter": 1,
-                              "last_parameter": 1
-                           }
-                        ]
-                     }
-                  },
+                  "<HAS>",
```

Comment:
I don't understand the removal of these, what is the reason?

Reply:
This is a test for the Refactoring Move Parameter tool, so it's better to just expect messages related to it.
The reason it's better is because otherwise, every time we add a refactoring tool or simply a code action (in this case, we're adding the Add Parameter tool), we would need to add messages here, affecting a test that does not have anything to do we the new functionality we're adding.